region/lowernorfair/east/Lower Norfair Spring Ball Maze Room.json

```diff
@@ -496,6 +507,10 @@
       "exitCondition": {
         "leaveWithSpark": {}
       },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
```

Comment:
are supers usable here?

Reply:
It's easier with a super imo.
Exiting `high` looks pretty impossible without screw though.  And if you carry the spark through the dragon with screw then it's not much of a hero shot anymore either.